addresses/mainnet.json

```diff
@@ -1213,6 +1213,17 @@
         "registryIds": [],
         "history": []
     },
+    {
+        "name": "AaveClaimStkAave",
+        "address": "0x5Dcd9Dc0185a6Ab07a31e5284D16Ce9f0A79Ce99",
+        "id": "0xd93d7e7f",
+        "path": "contracts/actions/aave/AaveClaimStkAave.sol",
+        "version": "1.0.0",
+        "inRegistry": true,
```

Comment:
It's not in registry, set to false, and changeTime to 0

Reply:
Fixed